pontoon/base/management/commands/update_auth_providers.py

```diff
@@ -10,42 +10,47 @@
 from django.contrib.sites.models import Site
 from allauth.socialaccount.models import SocialApp
 from allauth.socialaccount.providers.fxa.provider import FirefoxAccountsProvider
-from allauth.socialaccount.providers.github.provider import GithubProvider
+from allauth.socialaccount.providers.github.provider import GitHubProvider
 
 
 FXA_PROVIDER_ID = FirefoxAccountsProvider.id
-GITHUB_PROVIDER_ID = GithubProvider.id
+GITHUB_PROVIDER_ID = GitHubProvider.id
 
 
 class Command(BaseCommand):
     help = ('Ensures an allauth application exists and has credentials that match settings')
 
     def update_provider(self, data):
+        # Update the existing provider with current settings.
         try:
-            # Update the existing provider with current settings.
             app = SocialApp.objects.get(provider=data['provider'])
-            self.stdout.write("Updating existing authentication provider (pk=%s)" % app.pk)
             for k, v in data.items():
                 setattr(app, k, v)
+            self.stdout.write("Updating existing authentication provider (pk=%s)" % app.pk)
```

Comment:
Nit: After `save()` please, and then you can say "Updated" instead of "Updating" :)